### PR TITLE
docs(javascript): Add CDN bundle example for performance

### PR DESCRIPTION
**Pull request by benvinegar**
Instructions are now tabbed between ESM and CDN.

### CODE REVIEW

File: src/docs/platforms/javascript/index.mdx [hunk 1]
```diff
@@ -122,23 +122,27 @@ Upload source maps using either our [Webpack plugin](./config/sourcemaps/#webpac
...
 # Using yarn
 $ yarn add @sentry/browser @sentry/tracing
 
 # Using npm
 $ npm install @sentry/browser @sentry/tracing
 ```
 
-Alternatively, instead of `npm` packages, you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/tracing`:
-
-<JsCdnTag apm={true} />
```
Comment (benvinegar) [inline src/docs/platforms/javascript/index.mdx:137]: Wasn't sure how to get the tabbed experience while using this `<JsCdnTag/>` component. Open to ideas if someone can direct me.

**Review by rhcarvalho — APPROVED** [review]
LGTM.

Haven't tried using tags+tabs yet, maybe Cramer will be best bet if there's a better way to do it.

I noticed the Vercel deployment is kinda broken, every page loads and after a quick blink renders nothing but `Loading (StaticQuery)`. Guessing it is unrelated to the changes here.

File: src/docs/platforms/javascript/index.mdx (hunk 1 above)
Comment (dcramer) [reply src/docs/platforms/javascript/index.mdx:137]: for anyone else: you cant, what ben did here is the right solution

**Comment by dcramer** [comment]
im gonna update this and merge

**Pull request merged**